src/content/docs/en/guides/astro-db.mdx

```diff
@@ -815,3 +819,31 @@ Additionally, [pushing any table schema changes](#pushing-table-schemas) (also k
 :::danger
 If you override your `.db` file on deployment, you will lose your production data. Follow the deployment method process for your host carefully to prevent data loss.
 :::
+
+## Migrate from Astro Studio to Turso
```

Comment:
Possibly this section would be better under the "Astro Studio" section above? Especially given this will most likely be a relatively small group of people who had access anyway.

Reply:
Would approve that! I'll leave it to Otterlord to make the call, since we know we'll want to update again soon.

Reply:
sounds good to me